src/snovault/tests/test_searches_queries.py

```diff
@@ -2928,6 +2900,16 @@ def test_searches_queries_abstract_query_factory_add_source_object(dummy_request
     assert all([e in actual for e in expected])
     assert len(expected) == len(actual)
 
+def test_searches_queries_abstract_query_factory_add_exact_counting(params_parser, mocker):
+    from snovault.elasticsearch.searches.queries import AbstractQueryFactory
+    aq = AbstractQueryFactory(params_parser)
+    mocker.patch.object(AbstractQueryFactory, '_get_index')
+    AbstractQueryFactory._get_index.return_value = 'snovault-resources'
+    aq = AbstractQueryFactory(params_parser)
+    aq.add_source()
```

Comment:
Can get rid of this part, only needed for `add_source()`.

Reply:
sure thing, we will loose the first assertion, which is probably less important than it looks